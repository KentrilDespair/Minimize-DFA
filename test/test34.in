0,1,2,3,4,5
ab
0
2,4
0,a,1
0,b,1
1,a,5
1,b,2
2,a,3
2,b,3
3,a,4
3,b,5
4,a,5
4,b,5
