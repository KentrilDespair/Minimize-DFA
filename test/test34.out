0,1,2,3,4,5
ab
0
3,5
0,a,1
0,b,1
1,a,2
1,b,3
2,a,2
2,b,2
3,a,4
3,b,4
4,a,5
4,b,2
5,a,2
5,b,2
